utilities/src/main/java/io/syndesis/qe/cucumber/FeaturesInformationStore.java

```diff
@@ -0,0 +1,34 @@
+package io.syndesis.qe.cucumber;
+
+import java.net.URI;
+import java.util.HashMap;
+import java.util.Map;
+
+import io.cucumber.messages.Messages;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Storage for information usage in Formatters. The storage are fill up in onTestSourceRead method.
+ */
+@Slf4j
+public class FeaturesInformationStore {
```

Comment:
can't this be `@Component` and then `@Autowired`?

Reply:
if not, then it seems to me that is is not needed for this class to be singleton anyway :thinking: it is?

Reply:
I don't use that annotations because the store is not exist (NPE) in time when the cucumber plugin is being initialized.
As representation of some store of static information about all features file, i would say it should exist only once.